src/sql/workbench/api/common/sqlExtHostTypes.ts

```diff
@@ -1014,4 +1016,10 @@ export namespace designers {
 		Remove = 1,
 		Update = 2
 	}
+
+	export enum TableSubType {
```

Comment:
Is this mapped to anything in STS? Or where are these values coming from?

Reply:
Yes, they come from STS as stated in the PR description

Reply:
Would suggest providing a link to the relevant code in a comment (and ideally do the same in STS) so that it's easier to ensure they're kept in sync.